app/NotifyModule.scala

```diff
@@ -29,8 +34,39 @@ class NotifyModule extends Module {
       bind[NotificationService].to[HttpNotificationService],
       bind[MessageParser].to[S3EventParser],
       bind[DownloadUrlGenerator].to[S3DownloadUrlGenerator],
-      bind[QueueConsumer].to[SqsQueueConsumer],
-      bind[PollingJob].to[NotifyOnSuccessfulUploadProcessingFlow],
+      bind[PollingJobFactory].toProvider[PollingJobFactoryProvider],
       bind[ContinousPoller].toSelf.eagerly()
     )
 }
+
+class PollingJobFactoryProvider @Inject()(
+  successfulSqsQueueConsumer: SuccessfulSqsQueueConsumer,
+  quarantineSqsQueueConsumer: QuarantineSqsQueueConsumer,
+  successfulMessageParser: MessageParser,
```

Comment:
Why there are two MessageParsers injected here? Is there any difference between them?